app/controllers/HasCommodityCodeChangeController.scala

```diff
@@ -55,9 +55,14 @@ class HasCommodityCodeChangeController @Inject() (
   def onPageLoad(mode: Mode, recordId: String): Action[AnyContent] =
     (identify andThen profileAuth andThen getData andThen requireData).async { implicit request =>
       val record = goodsRecordConnector.getRecord(request.eori, recordId)
-
       record
         .map { goodsRecord =>
+          auditService.auditStartUpdateGoodsRecord(
```

Comment:
we need to remove auditing logic from CommodityCodeController otherwise it will audit log twice.